openshift/machine-config-operator
# OCPNODE-1416 : CGroupsV2 feature GA in MCO

1. The feature to enable CGroupsV2 on the nodes has been introduced in the OCP 4.11 as a `TechPreview` feature.
2. In the process of making it to GA, this code removes the `TechPreview` related checks.

Signed-off-by: Sai Ramesh Vanka <svanka@redhat.com>



**- What I did**
Removed the TechPreview related checks to enable CGroupsV2 version in the MCO's Kubelet Config Controller.
**- How to verify it**
Update the `nodes.config` CR's `cgroupMode` field to `v2` without the creation of a `TechPreviewNoUpgrade` featuregate and observe that the nodes are updated with the "CGroupsV2" configuration

**- Description for the changelog**

Make the enabling of the CGroupsV2 feature GA in MCO
